app/workers/postcodes_collection_worker.rb

```diff
@@ -0,0 +1,13 @@
+class PostcodesCollectionWorker < ApplicationWorker
+  sidekiq_options queue: :collect_postcodes
+
+  def perform
+    run_with_advisory_lock(Postcode, postcode) do
+      postcodes = Postcode.select(:postcode)
+
+      postcodes.each do |postcode|
+        ProcessPostcodeWorker.perform_async_in_queue(postcode, queue: :process_postcode)
```

Comment:
I think this will have the effect of immediately putting 12 million workers on a queue (once the database has been populated with all postcodes). How quickly this queue is processed is then down to each individual worker. For example, if the `ProcessPostcodeWorker` only takes 10ms to process each postcode, we'll work through 100 workers in a second, and could very quickly hit a rate limit on OS Places API (or worse, bring it down!).

I think this is the point where we need to artificially slow things down, probably by adding a `sleep 1` after line 9, so that only 1 worker is created every second. This keeps us safely in the limits of what is accepted by OS Places API.